src/app/drivingLicence/controllers/root.js

```diff
@@ -6,7 +6,7 @@ var logger = require('hmpo-logger').get();
 class RootController extends BaseController {
   async saveValues(req, res, next) {
     const sharedClaims = req.session?.shared_claims;
-    console.log('JB!@£$%^&*SharedClaims' + sharedClaims)
+    logger.info('JB!@£$%^&*SharedClaims' + sharedClaims)
```

Comment:
Again probably pointing out the obvious but feel free to get rid of my weird logging